src/components/Heading/__stories__/Heading.stories.mdx

```diff
@@ -0,0 +1,269 @@
+  Please check out our{" "}
+  <Link href="/?path=/docs/inputs-editableheading--overview" withoutSpacing size={Link.sizes.SMALL}>
+    EditableHeading
+  </Link>{" "}
+  component if you would like to allow users to edit the title text.
+</Tip>
+
+## Variants
+
+### Types and weights
+
+Heading component comes in three types: H1 (32px), H2 (24px), H3 (18px) and three weights: bold (700), normal (500), light (300)
+
+<Canvas>
+  <Story name="Types and weights">
+    <Flex
+      gap={Flex.gaps.SMALL}
+      direction={Flex.directions.COLUMN}
+      justify={Flex.justify.START}
+      align={Flex.align.START}
+    >
+      <Heading type={Heading.types.H1} weight={Heading.weights.BOLD}>
+        Bold H1 title
+      </Heading>
+      <Heading type={Heading.types.H1} weight="normal">
+        Normal H1 title
+      </Heading>
+      <Heading type={Heading.types.H1} weight={Heading.weights.LIGHT}>
+        Light H1 title
+      </Heading>
+      <Heading type={Heading.types.H2} weight={Heading.weights.BOLD}>
+        Bold H2 title
+      </Heading>
+      <Heading type={Heading.types.H2} weight={Heading.weights.NORMAL}>
+        Normal H2 title
+      </Heading>
+      <Heading type={Heading.types.H2} weight={Heading.weights.LIGHT}>
+        Light H2 title
+      </Heading>
+      <Heading type={Heading.types.H3} weight={Heading.weights.BOLD}>
+        Bold H3 title
+      </Heading>
+      <Heading type={Heading.types.H3} weight={Heading.weights.NORMAL}>
+        Normal H3 title
+      </Heading>
+      <Heading type={Heading.types.H3} weight={Heading.weights.LIGHT}>
+        Light H3 title
+      </Heading>
+    </Flex>
+  </Story>
+</Canvas>
+
+### Colors
+
+Heading component comes in four colors: primary, secondary, on primary, on inverted
+
+<Canvas>
+  <Story name="Colors">
+    <Flex direction={Flex.directions.COLUMN} align={Flex.align.START} gap={Flex.gaps.SMALL}>
+      <Heading type={Heading.types.H2} color={Heading.colors.PRIMARY}>
+        Primary title
+      </Heading>
+      <Heading type={Heading.types.H2} type={Heading.types.H2} type={Heading.types.H2} color={Heading.colors.SECONDARY}>
+        Secondary title
+      </Heading>
+      <Heading
+        element="div"
+        type={Heading.types.H2}
+        className={styles.primaryBackground}
+        align={Heading.align.CENTER}
+        color={Heading.colors.ON_PRIMARY}
+      >
+        On primary title
+      </Heading>
+      <Heading
+        element="div"
+        type={Heading.types.H2}
+        className={styles.invertedBackground}
+        align={Heading.align.CENTER}
+        color={Heading.colors.ON_INVERTED}
+      >
+        On inverted title
+      </Heading>
+    </Flex>
+  </Story>
+</Canvas>
+
+### Overflow
+
+Our Heading component supports overflow state.
+When the text is longer than its container and the ellipsis flag is on, the end of the text will be truncated and will display "..."
+
+<Canvas>
+  <Story name="Overflow" play={titleOverflowSuite}>
```

Comment:
Test is failing for me, please check that you see it failing in Chromatic as well